pkgdb2/lib/__init__.py

```diff
@@ -1259,9 +1270,10 @@ def edit_collection(session, collection, clt_name=None, clt_version=None,
             raise PkgdbException('Could not edit Collection.')
 
 
-def edit_package(session, package, pkg_name=None, pkg_summary=None,
-                 pkg_description=None, pkg_review_url=None,
-                 pkg_upstream_url=None, pkg_status=None, user=None):
+def edit_package(
+        session, package, pkg_name=None, pkg_summary=None,
+        pkg_description=None, pkg_review_url=None, pkg_upstream_url=None,
+        pkg_status=None, user=None):
```

Comment:
It's not needed here because we have the package object (rather than just its name), so the namespace is already attached and indeed, we cannot/don't want to allow editing it.